reana-cwl.yaml

```diff
@@ -0,0 +1,28 @@
+version: 0.1.0
+metadata:
+  authors:
+   - Harri Hirvonsalo <hjhsalo@gmail.com>
+   - Anton Khodak <anton.khodak@ukr.net>
```

Comment:
Please put Anton first; we are usually using alphabetical order everywhere.

Reply:
(based on alphabetical sorting of first names, like in REANA's main [AUTHORS.rst](https://github.com/reanahub/reana/blob/master/AUTHORS.rst))

Reply:
I thought it was sorted by last names :)